package.json

```diff
@@ -112,6 +113,7 @@
     "get-port": "5.1.1",
     "glob": "7.1.6",
     "gzip-size": "5.1.1",
+    "html-validator": "5",
```

Comment:
Any reason this isn't using the latest version, 6?

Reply:
Yes, v6 dropped support for Node.js v12 (ref: https://github.com/zrrrzzt/html-validator/commit/c3a11fa524ebe6e9822c0f219fcc83eaaa710853), hence failing the test using Node.js v12.
But Next.js support Node.js >= 12.22.0.

But yes instead to use `5`, we could use `5.1.18`. :+1: